commands.json

```diff
@@ -2815,7 +2815,8 @@
         "type": "enum",
         "enum": [
           "NX",
-          "XX"
+          "XX",
+          "GET"
         ],
```

Comment:
doesn't that mean that they're all mutually exclusive?
but in fact we do allow GET together with XX, so i gues it needs a separate block like the KEEPTTL has (and possibly a comment in the command's page that mention  that GET is incompatible with NX)

Reply:
Done!